cds_migrator_kit/rdm/records/transform/transform.py

```diff
@@ -127,7 +126,36 @@ def _media_files(self, entry):
         return {}
 
     def _pids(self, json_entry):
-        return json_entry.get("_pids", {})
+        from flask import current_app
+
+        DATACITE_PREFIX = current_app.config["DATACITE_PREFIX"]
+
+        pids = json_entry.get("_pids", {})
+        output_pids = deepcopy(pids)
+        for key, identifier in pids.items():
+            # ignoring some pids
+            if key.upper() in ["HAL", "HDL"]:
+                del output_pids[key]
```

Comment:
Why do we delete this? to avoid raising?

Reply:
because, we decided not to host some schemes as identifiers, so we transform it here